rdbms/src/main/java/org/spine3/server/storage/jdbc/aggregate/query/CreateMainTableQuery.java

```diff
@@ -31,6 +31,11 @@
 import static java.lang.String.format;
 import static org.spine3.server.storage.jdbc.aggregate.query.Constants.*;
 
+/**
+ * Query that creates a new aggregate main table if it does not exist.
+ *
+ * @author Andrey Lavrov
```

Comment:
Authors are defined in the chronological order. Yeah, there are a lot of questions about using @author, for example: how many lines should the person add/edit/remove to add that he is the author? and etc.